app/views/orga/projects/index.html.slim

```diff
@@ -10,7 +10,7 @@ table.table.table-striped.table-bordered.table-condensed.hidden-sm.hidden-xs
       th Submitted by
       th State
       th Tags
-      th colspan=3
+      th colspan=5
```

Comment:
I wonder if we should display the state-related actions all in the same column; there will only ever be one button to change the state of the project record, so we might as well have them in the same `<td>` all the time. What do you think?